jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/MetaInfConfiguration.java

```diff
@@ -174,7 +176,8 @@ public void findAndFilterContainerPaths(final WebAppContext context) throws Exce
         {
             List<Path> matchingBasePaths =
                 Stream.of(modulePath.split(File.pathSeparator))
-                    .map(URIUtil::toURI)
+                    .map(resourceFactory::newResource)
+                    .map(Resource::getURI)
                     .filter(uriPatternPredicate)
                     .map(Paths::get)
                     .toList();
```

Comment:
Yu'd think!  But because an exception is thrown, then forEach get's upset.   Not hot path, so happy to have the list and not clutter with a try catch.

Reply:
I meant calling `Stream.iterator()` followed by a classic `while (it.hasNext())` loop. But yes, that code path isn't perf critical so we can afford a little garbage.

Reply:
In ee10 I've done a better cleanup where I can use forEach and not convert resource to path and then back to resource.